kolibri/plugins/management/assets/src/vue/user-page/user-edit-modal.vue

```diff
@@ -2,22 +2,47 @@
 
   <div class="user-edit-modal">
     <modal btntext="Edit">
-      <div class="title" slot="header">User edit</div>
+
+      <h1 slot="header" class="header">Account Info</h1>
```

Comment:
@radinamatic in the mock-up, it said "Account Info" so it was changed. @whitzhu any thoughts?